src/scrabble.js

```diff
@@ -1,5 +1,77 @@
-function scrabble() {
-  // write code here
+  function scrabble() {
+    // write code here
+  // 1. Assigning the letters to scores
+  const letterScore = {
+    A: 1,
+    E: 1,
+    I: 1,
+    O: 1,
+    U: 1,
+    L: 1,
+    N: 1,
+    R: 1,
+    S: 1,
+    T: 1,
+    D: 2,
+    G: 2,
+    B: 3,
+    C: 3,
+    M: 3,
+    P: 3,
+    F: 4,
+    H: 4,
+    V: 4,
+    W: 4,
+    Y: 4,
+    K: 5,
+    J: 8,
+    X: 8,
+    Q: 10,
+    Z: 10
+  }
+  
+  // 2. Create a function called 'scrabble' with one parameter called 'word'
+  function scrabble(word) {
```

Comment:
You are copying the scrabble function within another scrabble function. This means you are not returning anything when `scrabble()` is invoked from outside this file. This is why all your tests are failing. Please update so that you have only one `scrabble()` function and only one module export